### DISCUSSION TITLE
Adonis js 5 deployment on vercel

### DISCUSSION THREAD

**Discussion by devKereya** [vercel/community#4033] [Help] [answered]
### Summary

I am tryng to deploy my adonis js api on vercel but there not found my api 404: NOT_FOUND
Code: NOT_FOUND

### Example

https://server-kereya-6mti2cr58-devkereya.vercel.app/

### Steps to Reproduce

<img width="672" alt="Capture d’écran 2023-09-12 à 11 23 10" src="https://github.com/vercel/community/assets/125360560/0c2fb8c5-405c-42c6-b2ec-c368b048d71c">
<img width="980" alt="Capture d’écran 2023-09-12 à 11 23 34" src="https://github.com/vercel/community/assets/125360560/9855a966-8091-4199-9210-955860a10c3b">
<img width="1012" alt="Capture d’écran 2023-09-12 à 11 23 57" src="https://github.com/vercel/community/assets/125360560/ed35870b-0bb9-4eae-a79f-39963aad7691">

**Comment by amyegan** [comment, marked as answer]
Hey @devKereya. You can check the build source structure and output by going to the deployment's details page, then the `source` tab on that page, then toggle between "source" and "output" on that page. That may give you some clues about what is gong wrong.

If you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will give us a better view of what is going wrong so we can help with debugging.

**Reply by architkumar314** [reply]
Can u please help me to deploy adonis js application on vercel.
